azext_edge/edge/util/az_client.py

```diff
@@ -141,6 +144,24 @@ class DeviceRegistryMgmtApiVersion(Enum):
     V20240901_preview = "2024-09-01-preview"
 
 
+def get_registry_refresh_mgmt_client(
```

Comment:
This should no longer be necessary, get_registry_mgmt_client should cover the new API